src/lightning_app/cli/lightning_cli.py

```diff
@@ -116,6 +121,51 @@ def run_app(
     _run_app(file, cloud, without_server, no_cache, name, blocking, open_ui, env)
 
 
+@run.command("command")
+@click.argument("command", type=str, default="")
+@click.option(
+    "--args",
+    type=str,
+    default=[],
+    multiple=True,
+    help="Arguments to be passed to the method executed in the running app.",
```

Comment:
There is no argument parsing for "method arguments". So the only allowed arguments would be string. 
I recommend not adding args at all in the minimal version of this feature. In any case, where is the documentation? How should the user parse these inputs?

Reply:
Added support for some validation layers, especially with ClientCommand.